unlock-protocol.com/blog/unlock-this-blog.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Unlock this blog
+subTitle:
+authorName: Julien Genestoux
+publishDate: July 24, 2019
+description:
+image:
+---
+
+Unlock is a [protocol for memberships](/blog/protocol-for-membership). It helps creators monetize their community (fans, supporters, or even simple users) by helping them deploy a lock and sell access keys to this lock (they can chose their currency, price, the duration of the keys and the number of keys they want to sell...).
```

Comment:
I think that we may want to rephrase this a bit ` It helps creators monetize their community` as the language implies they are monetizing access to the audience as opposed to the content/creation/access etc